tasks/setup-Debian.yml

```diff
@@ -12,3 +12,9 @@
     repo: 'deb https://packages.elastic.co/kibana/{{ kibana_version }}/debian stable main'
     state: present
     update_cache: yes
+
+- name: Create kibana config directory
```

Comment:
Since this could affect RedHat as well, and is safe to run regardless, can you stick this here, in the main setup file? https://github.com/geerlingguy/ansible-role-kibana/blob/f3c0766f5b4b6968da97fc6da7b48562dca3663e/tasks/main.yml#L17